protected/humhub/events/MigrationEvent.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * @link      https://www.humhub.org/
+ * @copyright Copyright (c) 2023 HumHub GmbH & Co. KG
+ * @license   https://www.humhub.com/licences
+ */
+
+namespace humhub\events;
+
+use humhub\components\Event;
+use humhub\interfaces\ApplicationInterface;
+use yii\base\Module;
+
+/**
+ * @property int|null $result Result of the migration:
+ * ``
+ *  - `ExitCode::OK`: Success;
+ *  - `ExitCode::UNSPECIFIED_ERROR`: failure;
+ *  - `Null`: nothing done
+ * ``
+ */
+class MigrationEvent extends Event
+{
+    /**
+     * @var \humhub\components\Module|ApplicationInterface|null
+     */
+    public ?Module $module;
+
+    /**
+     * @var string|null path to the migrations folder for modules, empty for application
+     */
+    public ?string $path;
```

Comment:
Can be removed?

Reply:
nope. But is private now. With private getter only. And `$config` has also been removed. So this should do no harm.